src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -0,0 +1,29 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var BrambleEvents = brackets.getModule("bramble/BrambleEvents");
+    var EditorManager = brackets.getModule("editor/EditorManager");
+    var CommandManager = brackets.getModule("command/CommandManager");
+    var Commands = brackets.getModule("command/Commands");
+
+    var ConsoleManagerRemote = require("text!lib/ConsoleManagerRemote.js");
+    var IFrameBrowser = require("lib/iframe-browser");
+
+    var isConsoleEnabled = false;
+
+    function getRemoteScript() {
+        return "<script>\n" + ConsoleManagerRemote + "</script>\n";
```

Comment:
Thank you @humphd for the clarification, I had saw 

So, revert back to the implementation in the current PR?